input/pagecontent/general-guidance.md

```diff
@@ -181,32 +178,24 @@ Example Condition - Cellulitis of right knee
       },
       "text" : "Cellulitis of right knee"
     ]
```

Comment:
Need to add elipse ... at the end of the snipped - you can see what they looks like in example snippet here https://build.fhir.org/ig/hl7au/au-fhir-core/general-requirements.html#missing-must-support-and-mandatory-data